lib/modules/manager/gradle/parser.ts

```diff
@@ -18,6 +19,8 @@ import { isDependencyString, parseDependencyString } from './utils';
 
 const groovy = lang.createLang('groovy');
 
+setParseGradleFunc(parseGradle);
```

Comment:
Is it possible to make another configuration that avoids `setSomething` functions? I'm okay with it temporarily, but maybe let's add something like `TODO: reorganize avoiding cycles` for further refactoring.

Reply:
```suggestion
// TODO: reorganize to avoide cycles
setParseGradleFunc(parseGradle);
```

It would require a lot changes to reorganize and I'm probably not the best developer to fix that 😉 
So let's do this quick fix and you can refactor it later

Reply:
Great, that's all I wanted :)